cpp/models/secir/secir_parameters_io.h

```diff
@@ -161,28 +168,28 @@ IOResult<Graph<Model, MigrationParameters>> read_graph(const std::string& direct
  * @param tmax end point of simulation
  */
 template <class Simulation>
-IOResult<void> write_single_run_result(const int run, const epi::Graph<epi::SimulationNode<Simulation>, epi::MigrationEdge>& graph)
+IOResult<void> write_single_run_result(const int run, const mio::Graph<mio::SimulationNode<Simulation>, mio::MigrationEdge>& graph)
```

Comment:
maybe I was unable to find it, but this function is actually never tested, is it?